### PR TITLE
fix: add telemetry to core charts

### PR DESCRIPTION
**Pull request by danielbdias**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: charts/tracetest-core/templates/configmap.yaml [hunk 1]
```diff
@@ -20,23 +20,28 @@ data:
...
+      {{- if .Values.config.telemetry.enabled }}
+      telemetry:
+        exporter: collector
+      {{- end }}
 
   provisioning.yaml: |-
       ---
       type: Config
       spec:
-        analyticsEnabled: false
```
Comment (jorgeepc) [inline charts/tracetest-core/templates/configmap.yaml:42]: Does this mean that we are not getting analytics?

**Review by jorgeepc — APPROVED** [review]

File: charts/tracetest-core/templates/configmap.yaml (hunk 1 above)
Comment (jorgeepc) [reply charts/tracetest-core/templates/configmap.yaml:42]: Nee I don't think we control analytics with this, right?

**Review by xoscar — APPROVED** [review]

File: charts/tracetest-core/templates/configmap.yaml (hunk 1 above)
Comment (mathnogueira) [reply charts/tracetest-core/templates/configmap.yaml:42]: No, use use another flag.

**Review by mathnogueira — APPROVED** [review]

File: charts/tracetest-core/templates/configmap.yaml (hunk 1 above)
Comment (mathnogueira) [reply charts/tracetest-core/templates/configmap.yaml:42]: No, we use another flag.

**Pull request merged**